src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -617,7 +632,7 @@ protected DynamoModel(IStartConfiguration config)
                     }
                     catch (Exception ex)
                     {
-                        Logger.Log(ex.Message);                       
```

Comment:
Visual Studio stripping trailing whitespace.
Add w=1 to the query part of the GitHub URL to ignore whitespace changes.
This usually means just adding ?w=1, but may need to be &w=1 if there's already a ?.